### PR TITLE
[Merged by Bors] - feat(SetTheory/Ordinal/Exponential): `lt_omega0_opow`

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Ordinal/Exponential.lean [hunk 1]
```diff
@@ -479,6 +479,23 @@ theorem add_log_le_log_mul {x y : Ordinal} (b : Ordinal) (hx : x ≠ 0) (hy : y
...
+  use n.succ
+  rw [natCast_succ, ← hn]
+  exact lt_mul_succ_div a (opow_ne_zero _ omega0_ne_zero)
+
+theorem lt_omega0_opow_succ {a b : Ordinal} (ha : a < ω ^ succ b) : ∃ n : ℕ, a < ω ^ b * n := by
+  obtain ⟨c, hc, n, hn⟩ := lt_omega0_opow ha (succ_ne_zero b)
+  refine ⟨n, hn.trans_le (mul_le_mul_right' ?_ _)⟩
+  rwa [opow_le_opow_iff_right one_lt_omega0, ← lt_succ_iff]
+
+theorem omega0_opow_mul_nat_lt (a : Ordinal) (n : ℕ) : ω ^ a * n < ω ^ succ a := by
```
Comment (fpvandoorn) [inline Mathlib/SetTheory/Ordinal/Exponential.lean:495]:
```suggestion
theorem omega0_opow_mul_nat_lt {a b : Ordinal} (h : a < b) (n : ℕ) : ω ^ a * n < ω ^ b := by
```
I think this statement is more convenient

**Review by fpvandoorn — COMMENTED** [review]
Maybe all three of these should be `iff` statements?

File: Mathlib/SetTheory/Ordinal/Exponential.lean (hunk 1 above)
Comment (vihdzp) [reply Mathlib/SetTheory/Ordinal/Exponential.lean]: I agree!

**Comment by fpvandoorn** [comment]
bors merge

**Pull request closed without merging**